<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="troy-PC" name="Default test" tests="1" failures="1" timestamp="8 Dec 2015 13:24:15 GMT" time="31.828" errors="0">
  <testcase name="test000login" time="26.539" classname="projects.ticket.testCases.ticketWeb.addBusPiaoJia">
    <failure type="java.lang.AssertionError" message="Can not find the 登录输入框 element on the 登录页 page.The css is #loginform &amp;gt; form &amp;gt; div:nth-child(1) &amp;gt; div &amp;gt; input">
      <![CDATA[java.lang.AssertionError: Can not find the 登录输入框 element on the 登录页 page.The css is #loginform > form > div:nth-child(1) > div > input
	at org.testng.Assert.fail(Assert.java:94)
	at test.autotest.core.UI.findElement(UI.java:219)
	at test.autotest.core.UI.findElement(UI.java:181)
	at test.autotest.core.UI.sendKeys(UI.java:45)
	at test.autotest.base.WebApp.runTestCase(WebApp.java:348)
	at projects.ticket.testCases.ticketWeb.addBusPiaoJia.test000login(addBusPiaoJia.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- test000login -->
</testsuite> <!-- Default test -->
